src/TimeoutMigrationTool.Raven4.AcceptanceTests/RavenDBToAsqEndToEnd.cs

```diff
@@ -0,0 +1,150 @@
+﻿namespace TimeoutMigrationTool.Raven4.AcceptanceTests
+{
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NUnit.Framework;
+    using Particular.TimeoutMigrationTool;
+    using Particular.TimeoutMigrationTool.RavenDB;
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NServiceBus.Features;
+    using Particular.TimeoutMigrationTool.ASQ;
+
+    [TestFixture]
+    class RavenDBToAsqEndToEnd : RavenDBAcceptanceTest
+    {
+        string asqConnectionString = Environment.GetEnvironmentVariable("AzureStorageQueue_ConnectionString") ?? "UseDevelopmentStorage=true";
+
+        [Test]
+        public async Task Can_migrate_timeouts()
+        {
+            var sourceEndpoint = NServiceBus.AcceptanceTesting.Customization.Conventions.EndpointNamingConvention(typeof(LegacyRavenDBEndpoint));
+            var targetEndpoint = NServiceBus.AcceptanceTesting.Customization.Conventions.EndpointNamingConvention(typeof(AsqEndpoint));
+
+            var ravenTimeoutPrefix = "TimeoutDatas";
+            var ravenVersion = RavenDbVersion.Four;
+
+            var ravenAdapter = new Raven4Adapter(serverUrl, databaseName);
+
+            await Scenario.Define<SourceTestContext>()
+                .WithEndpoint<LegacyRavenDBEndpoint>(b => b.CustomConfig(ec =>
+                    {
+                        ec.UsePersistence<RavenDBPersistence>()
+                            .SetDefaultDocumentStore(GetDocumentStore(serverUrl, databaseName));
+                        ec.UseSerialization<NewtonsoftSerializer>();
+                    })
+                    .When(async (session, c) =>
+                    {
+                        var delayedMessage = new DelayedMessage();
+
+                        var options = new SendOptions();
+
+                        options.DelayDeliveryWith(TimeSpan.FromSeconds(20));
+                        options.SetDestination(targetEndpoint);
+
+                        await session.Send(delayedMessage, options);
+
+                        await WaitUntilTheTimeoutIsSavedInRaven(ravenAdapter, sourceEndpoint);
+
+                        c.TimeoutSet = true;
+                    }))
+                .Done(c => c.TimeoutSet)
+                .Run(TimeSpan.FromSeconds(15));
+
+            var context = await Scenario.Define<TargetTestContext>()
+                .WithEndpoint<LegacyRavenDBEndpoint>(b => b.CustomConfig(ec =>
```

Comment:
Why is it necessary to bootstrap this one as well?

Reply:
Is it because of https://github.com/Particular/TimeoutMigrationTool/pull/171/files#diff-e74c55f9dbb0e389c88bfe007b80d5d37726fe69b7069684a973434d1136b03aR70-R72?

Reply:
Yes, I'll add the comment here as well